lib/council-server/src/lib.rs

```diff
@@ -80,5 +94,13 @@ pub enum Response {
     BeenProcessed { node_id: Id },
     Failed { node_id: Id },
     OkToProcess { node_ids: Vec<Id> },
+    Restart,
     Shutdown,
 }
+
+#[remain::sorted]
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(tag = "kind")]
+pub enum ManagementResponse {
+    Restart,
+}
```

Comment:
Both here and in the enums above, we have added the ability for `council` to tell all or individual `pinga` instances to restart actively running jobs